plugins/asdf/asdf.plugin.sh

```diff
@@ -0,0 +1,30 @@
+#! bash oh-my-bash.module
+# asdf.plugin.sh: asdf plugin for oh-my-bash.
+# Author: @RobLoach (https://github.com/robloach)
+# Author: @prodrigues1912 (https://github.com/prodrigues1912)
+# Originally suggested in https://github.com/ohmybash/oh-my-bash/pull/310
+# Fork of the oh-my-zsh asdf plugin
+
+# Custom ASDF_DIR location
+if [[ ${ASDF_DIR+set} ]]; then
+  . "$ASDF_DIR/asdf.sh"
+
+# Standard install location
+elif [[ -f "${XDG_CONFIG_HOME:-$HOME}/.asdf/asdf.sh" ]]; then
+  ASDF_DIR="${XDG_CONFIG_HOME:-$HOME}/.asdf"
```

Comment:
Is this location correct? `XDG_CONFIG_HOME` is defaulted to `$HOME/.config`. It's not normal to have a dotdir under `~/.config` (i.e. `~/.config/.asdf`). The global code search in GitHub only finds only your setting:

- [Code search results `/${XDG_CONFIG_HOME:-$HOME}\/.asdf/`](https://github.com/search?q=%2F%24%7BXDG_CONFIG_HOME%3A-%24HOME%7D%5C%2F.asdf%2F&amp;type=code)

The upstream discussions (https://github.com/asdf-vm/asdf/issues/687, https://github.com/asdf-vm/asdf/pull/1351) .only mention `$XDG_CONFIG_HOME/asdf`

Reply:
Perhaps something like this?
``` suggestion
# Home
elif [[ -f "$HOME/.asdf/asdf.sh" ]]; then
  ASDF_DIR="$HOME/.asdf"
  . "${ASDF_DIR}/asdf.sh"

# Config
elif [[ -f "$XDG_CONFIG_HOME/asdf/asdf.sh" ]]; then
  ASDF_DIR="$XDG_CONFIG_HOME/asdf"
```

Reply:
That works, but please replace `$XDG_CONFIG_HOME` with `${XDG_CONFIG_HOME:-$HOME/.config}`.